date_range/wizard/date_range_generator.py

```diff
@@ -43,14 +43,17 @@ def _compute_date_ranges(self):
                      count=self.count+1)
         vals = list(vals)
         date_ranges = []
+        month_digits = len(unicode(self.count))
         for idx, dt_start in enumerate(vals[:-1]):
             date_start = fields.Date.to_string(dt_start.date())
             # always remove 1 day for the date_end since range limits are
             # inclusive
             dt_end = vals[idx+1].date() - relativedelta(days=1)
             date_end = fields.Date.to_string(dt_end)
             date_ranges.append({
-                'name': '%s-%d' % (self.name_prefix, idx + 1),
+                'name': '%s-%s' % (
```

Comment:
That was needed indeed. While you are at it could you remove the `-`. Sometimes we don't want a dash, or we want something else, so it's better to let the user add it in the prefix.